.github/workflows/build.yml

```diff
@@ -0,0 +1,76 @@
+name: glvd
+
+on:
+  pull_request:
+  release:
+    types:
+    - published
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Build and export
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: Containerfile
+          tags: glvd:latest
+          outputs: type=docker,dest=/tmp/glvd.tar
+
+      - name: Upload artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: glvd
+          path: /tmp/glvd.tar
+          retention-days: 7
+
+  test:
```

Comment:
One job, except this does not work.

Reply:
I have seen your change and now I got it.